WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -3,59 +3,217 @@ import SwiftUI
 struct TotalsView: View {
     @ObservedObject private var viewModel: PointOfSaleDashboardViewModel
 
+    enum PaymentState {
```

Comment:
Should we make the `PaymentState` be held by `PointOfSaleDashboardViewModel`? At the moment it's just  internal state for the View. It also makes sense to move it if we're expecting it to change upon different button tapping or cart state updates from somewhere else.